### PR TITLE
Isolate tests

### PR DESCRIPTION
**Pull request by joeldrapper**
What you gon' do when I appear?
W-W-when I premiere?
to isolate Rails when it shouldn't be there?

I was in the #212.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -20,8 +20,11 @@ jobs:
           ruby-version: ${{ matrix.ruby-version }}
           bundler-cache: true
 
-      - name: Tests
-        run: bundle exec sus
+      - name: Isolated tests
+        run: bundle exec sus test/isolated/**/*rb
+
+      - name: Rails tests
+        run: bundle exec sus test/rails/**/*rb
```
Comment (ParamagicDev) [inline .github/workflows/ci.yml:27]: I think this glob is missing a file extension separator.

```suggestion
      - name: Isolated tests
        run: bundle exec sus test/isolated/**/*.rb

      - name: Rails tests
        run: bundle exec sus test/rails/**/*.rb
```

Comment (joeldrapper) [reply .github/workflows/ci.yml]: I think the issue is `**/*` is interpreted differently in GitHub actions than it is on my local machine. Locally, it means “in this folder and/or sub folders”. In GitHub actions, it means “in sub folders”.

Comment (ParamagicDev) [reply .github/workflows/ci.yml]: Perhaps this is a bash vs sh issue?

could try:

`/bin/bash -e "bundle exec sus test/isolated/**/*.rb"`

Comment (joeldrapper) [reply .github/workflows/ci.yml]: That doesn't work either. 😞

**Pull request closed without merging**